src/js/components/awardv2/visualizations/amounts/AggregatedAwardAmounts.jsx

```diff
@@ -49,10 +50,20 @@ export default class AggregatedAwardAmounts extends React.Component {
             const awardAmounts = this.props.awardAmounts;
             const visualizationType = determineScenario(awardAmounts);
```

Comment:
Wonder if maybe we could change this `fn` name at some point to be more descriptive for readability. IE `determineAmountSpent`?

Reply:
Renamed to `determineSpendingScenario`.

Reply:
👌